demo/web/index.html

```diff
@@ -0,0 +1,221 @@
+      }
+
+      controlBtnEl.addEventListener('click', () => {
+        if (!isPlaying) {
+          originalAudioSource = audioContext.createBufferSource();
+          originalAudioSource.addEventListener('ended', onAudioStop);
+          originalAudioSource.buffer = createBuffer(pcm);
+          originalAudioSource.connect(originalAudioGain);
+          originalAudioSource.start();
+
+          isPlaying = true;
+          controlBtnEl.innerText = 'Stop';
+          textToSynthesizeEl.disabled = true;
+          speechRateSliderEl.disabled = true;
+          synthesizeBtnEl.disabled = true;
+        } else {
+          originalAudioSource.stop();
+          onAudioStop();
+        }
+      });
+    };
+
+    function orcaSpeechCallback(orcaSpeech) {
+      pcm = orcaSpeech.speech;
+      writeMessage('Synthesizing complete!');
+      document.getElementById('text-to-synthesize').disabled = false;
+      document.getElementById('speech-rate').disabled = false;
+      document.getElementById('control-btn').disabled = false;
+      document.getElementById('download-btn').disabled = false;
+    }
+
+    async function startOrca(accessKey) {
+      writeMessage('Orca is loading. Please wait...');
+      try {
+        document.getElementById('start-orca').disabled = true;
+        document.getElementById('text-to-synthesize').disabled = true;
+
+        orca = await OrcaWeb.OrcaWorker.create(
+          accessKey,
+          orcaSpeechCallback,
+          { base64: modelParams, forceWrite: true },
+          { synthesizeErrorCallback: (e) => writeMessage(e) },
+        );
+
+        document.getElementById('text-to-synthesize').disabled = false;
+        document.getElementById('speech-rate').disabled = false;
+        writeMessage('Orca worker ready!');
+      } catch (err) {
+        writeMessage(err);
+      }
+    }
+
+    async function synthesize() {
+      writeMessage('Synthesizing. Please wait...');
+      try {
+        document.getElementById('text-to-synthesize').disabled = true;
+        document.getElementById('speech-rate').disabled = true;
+        document.getElementById('synthesize-btn').disabled = true;
+        document.getElementById('control-btn').disabled = true;
+        document.getElementById('download-btn').disabled = true;
+        const text = document.getElementById('text-to-synthesize').value;
+        const speechRate = document.getElementById('speech-rate').value;
+        await orca.synthesize(text, speechRate);
+      } catch (err) {
+        writeMessage(err);
+      }
+    }
+
+    function downloadDumpAudio() {
+      let blob = new Blob([pcm]);
+      let a = document.createElement('a');
+      a.download = 'orca_speech_audio.pcm';
+      a.href = window.URL.createObjectURL(blob);
+      a.click();
+      document.removeChild(a);
+    }
+  </script>
+</head>
+<body>
+<h1>Orca Web Demo</h1>
+<p>This demo uses Orca for Web to:</p>
+<ol>
+  <li>
+    Create an instance of Orca with the model file provided.
```

Comment:
`with the model filed provided` -> `with our default voice`?

Reply:
hm I'm not sure, `the model file provided` refers to the actual model file in the script tag. I think this is more clear than "our default voice"?